services/core-web/src/components/mine/MineComments.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+
+import { getMineComments } from "@common/selectors/mineSelectors";
+import {
+  deleteMineComment,
+  createMineComment,
+  fetchMineComments,
+} from "@common/actionCreators/mineActionCreator";
+import CommentPanel from "@/components/common/comments/CommentPanel";
+
+const propTypes = {
+  mineGuid: PropTypes.string.isRequired,
+  comments: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.any)),
```

Comment:
I think we should make a custom prop type to use in these cases.

Reply:
ah you are correct. I'll add that